woocommerce/woocommerce-android
# Feature/add qr code scan on magic link sent

~~Do not merge until[ this PR](https://github.com/wordpress-mobile/WordPress-Login-Flow-Android/pull/102) is merged and the login library changeset updated in `build.graadle`~~

**EDIT:** This PR doesn't rely on changes from auth library anymore

### Description
Adds scan Qr code CTA to magic link email sent screen. Design based on @joe-keenan proposal here: p1667455482904769/1667406126.391389-slack-C03L1NF1EA3

### Testing instructions
- Click on `Continue with WordPress.com` button
- Enter an existing email
- Select "Log in with magic link" 
- Check that the new CTA to scan QR code is displayed following the design from the screenshot below
- Scan the code and check that user is logged in

### Images/gif

![ScanQrCta](https://user-images.githubusercontent.com/2663464/199758675-b5215759-ed3a-491e-bc55-5331fca135b1.png)
